pandas/tests/reshape/test_pivot.py

```diff
@@ -1638,3 +1639,31 @@ def test_crosstab_tuple_name(self, names):
 
         result = pd.crosstab(s1, s2)
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize("names", [['a', 'b'],
+                                       [('a', 'b'), 'c'],
+                                       [('a', 'b'), ('c', 'd')],
+                                       [(1, 2, 3), ('a', 'b', 'c')]])
+    def test_crosstab_cols_output(self, names):
+        rows = [[1, 2, 3, 4], [1, 1, 2, 2], [1, 3, 1, 4]]
+        cols = [[1, 1, 1, 1], [3, 2, 2, 3], []]
+
+        expected_ct1 = pd.DataFrame(
```

Comment:
put the expected IN the parameterize, e.g.

```
@pytest.mark.parametrize(
"names, expected",
[
([['a', 'b'], pd.DataFrame(......)),
(.....)
(.....)
])
```

Reply:
This might not be possible as for each "name", there are 3 different expected-s. The only parameter here is name. Am I missing anything?

Reply:
make that a parameter as well if needed
this is just a convenient way of writing out the cases

Reply:
Do you mean something like parametrize( "name, expected1, expected2, expected3", [...])?

Reply:
no

""name,input,expected", [(....), (...)]

Reply:
Uhm. Ok. Thanks for the pointer. I'm still new at purest. Will do that then.